Source/WebKit/WebProcess/cocoa/VideoFullscreenManager.mm

```diff
@@ -454,6 +464,16 @@ static FloatRect inlineVideoFrame(HTMLVideoElement& element)
 {
     if (m_page)
         m_page->send(Messages::VideoFullscreenManagerProxy::SetVideoDimensions(contextId, videoDimensions));
+
+    if (m_setupFullscreen) {
+        auto [model, interface] = ensureModelAndInterface(contextId);
+        RefPtr<HTMLVideoElement> videoElement = model->videoElement();
```

Comment:
auto videoElement

need null check

Reply:
Good catch! Fixed in latest patch.

Thanks for reviewing!